plsync/mlabconfig_test.py

```diff
@@ -515,18 +514,37 @@ def test_export_legacy_includes_selected_experiments(self):
         for hostname, node in self.sites[0]['nodes'].iteritems():
             experiments[0].add_node_address(node)
         output = StringIO.StringIO()
-        expected_config = {
-            u'labels': {u'service': u'sidestream'},
-            u'targets': [
-                u'bar.abc.mlab2.abc01.measurement-lab.org:9090'
-            ]
-        }
+        expected_targets = [
+            u'bar.abc.mlab2.abc01.measurement-lab.org:9090'
+        ]
 
-        mlabconfig.export_legacy(output, experiments, "bar.abc.mlab2.*")
+        actual_targets = mlabconfig.select_prometheus_experiment_targets(
+            experiments, "bar.abc.mlab2.*", '{{hostname}}:9090', False)
```

Comment:
Would it be worth modifying this test, or adding a new one, to test that experiments which _do not_ match the passed regex are in fact filtered as expected?